src/pudl/metadata/resources/eiaaeo.py

```diff
@@ -0,0 +1,514 @@
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__electric_sector_fuel_cost_by_type": {
+        "description": (
+            "Projected fuel prices to the electric power sector, including "
+            "electricity-only and combined-heat-and-power plants that have a "
+            "regulatory status."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_fuel_cost_type",
+                "real_value_base_year",
+                "fuel_cost_mmbtu",  # this is *nominal* dollars, but we probably want to standardize to some sort of constant-dollar value?
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_fuel_cost_type",
+                "real_value_base_year",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__electric_power_sector_emissions": {
+        "description": (
+            "Projected emissions from the electric power sector, including "
+            "electricity-only and combined-heat-and-power plants that have a "
+            "regulatory status."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "carbon_mass_tons",
+                "co2_mass_tons",
+                "hg_mass_lb",
+                "nox_mass_lb",
+                "so2_mass_lb",
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+}
+
+# 2024-04-24: A holding pen so it's easy to see what the field definitions are.
+# Once we "promote" schemas we need to move the fields to the real
+# pudl.metadata.fields.FIELD_METADATA dictionary. What a frickin pain.
+_STAGING_FIELD_METADATA: dict[str, dict[str, Any]] = {
+    "report_year": {
+        "type": "integer",
+        "description": "ALREADY EXISTS, HERE FOR CONVENIENCE!",
+    },
+    "projection_date": {
+        "type": "date",
+        "description": "Date at which this data is projected to be true.",
+    },
+    "real_value_base_year": {
+        "type": "integer",
+        "description": "Nominal monetary values are converted to values from this year.",
+    },
+    "electricity_market_module_region": {
+        "type": "string",
+        "description": "AEO projection region. Maybe the same as NERC region?",
```

Comment:
As mentioned in standup, here's a fun question... this seems similar to NERC region, are they the same **concept** or do they just happen to have similar values?